Source/deviceinfo/deviceinfo.h

```diff
@@ -24,7 +24,8 @@ typedef enum deviceinfo_hdcp_type {
     DEVICEINFO_HDCP_14,
     DEVICEINFO_HDCP_20,
     DEVICEINFO_HDCP_21,
-    DEVICEINFO_HDCP_22
+    DEVICEINFO_HDCP_22,
+    DEVICEINFO_HDCP_LENGTH
```

Comment:
Perhaps I miss something but why was this added?

Reply:
The other Enums had LENGTH as the last value, but only this one didn't have. Hence for uniformity's sake, I added it.

Reply:
Good reason, I like consistency ;)